tests/test_mistral_tokenizer.py

```diff
@@ -48,7 +48,7 @@ def test_from_model(self) -> None:
     def test_decode(self, special_token_policy: Optional[SpecialTokenPolicy], is_tekken: bool) -> None:
         tokenizer = MistralTokenizer.v3(is_tekken=is_tekken)
 
-        prompt = "This is a complicated te$t, ain't it?"
+        prompt = "This is a complicated test, ain't it?"
```

Comment:
```suggestion
        prompt = "This is a complicated te$t, ain't it?"
```

This is on purpose :)